gcloud-java-datastore/src/test/java/com/google/gcloud/datastore/LocalDatastoreHelperTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.gcloud.datastore;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertSame;
+import static org.junit.Assert.assertTrue;
+
+import com.google.gcloud.AuthCredentials;
+import com.google.gcloud.datastore.testing.LocalDatastoreHelper;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.io.IOException;
+
+@RunWith(JUnit4.class)
+public class LocalDatastoreHelperTest {
+
+  private static final String PROJECT_ID = "some-project-id";
+  private static final double TOLERANCE = 0.00001;
+
+  @Test
+  public void testStart() throws IOException, InterruptedException {
```

Comment:
If we follow my suggestion of splitting `start` into `create` and `start` this test should be split and `start/stop` would no longer be needed to test that fields have been correctly initialized.